### ISSUE TITLE
Page context menu not working with multi-process

### ISSUE THREAD

**Issue by sneakypete81** [sneakypete81/updatescanner#89]
From @keith94:
"Scan Page For Updates" is broken with multi-process.
```
TypeError: window.content is null[Learn More]  updatescanoverlay.js:58:8
TypeError: mainWindow.content is null[Learn More]  updatescan.js:284:9
```

**Comment by sneakypete81** [comment]
I think I have a fix for the context menu. But another issue is getting the URL/title of the current tab when clicking on this "New" button in the sidebar. I don't currently have a solution for this.

**Comment by sneakypete81** [comment]
Fixed in v3.3.2beta2.
